### DISCUSSION TITLE
Adding a minor feature

### DISCUSSION THREAD

**Discussion by 13K-0** [Alovoa/alovoa#235] [Show and tell]
Can you add a feature "Sign up with email" and two options "Signup" , "Login" on the Alovoa App listed or available on the Google Play Store. 
The feature is present on Alovoa Lite.
Thank you 😊

**Comment by Nonononoki** [comment]
Hi, 
the Android app has its own repo, please post any stuff related to the app to this repo: https://github.com/Alovoa/alovoa-expo
The current version in the Google Play Store is outdated, because the review team can take months to review an update. You can already login with your email address in the Alovoa app in the latest version, located here: https://github.com/Alovoa/alovoa-expo/releases/latest

**Comment by 13K-0** [comment]
At last today i have got the Alovoa login by email update which was pushed by the developer on 07/12/22 
Thank you bye the way Alovoa Developer ❤️

**Comment by 13K-0** [comment]
Please correct the word from "Sign in with Facebook to Sign in with Google' "

![Screenshot_2022-12-28-12-18-47-691-edit_com.alovoa.expo.jpg](https://user-images.githubusercontent.com/68194908/209770557-4cb6b01c-14c1-4913-888d-a4e9254bed25.jpg)
